blueprints/02-at-scale/main.tf

```diff
@@ -66,6 +66,15 @@ locals {
 
   cbci_agent_podtemplname_validation = "maven-and-go-ondemand"
 
+  global_password = random_string.global_pass_string.result
```

Comment:
Perhaps use a change trigger to allow someone to rotate the password every once in a while?

```terraform
variable "change_trigger" {
  description = "Change this value to generate a new random string"
  default     = "initial"
}

resource "random_string" "global_pass_string" {
  length  = 16
  special = false
  upper   = true
  lower   = true

  keepers = {
    change_trigger = var.change_trigger
  }
}
```

Reply:
@sboardwell Thanks for your review! :) 

Avoid hardcoding secrets is a must and it needs to be handle since once you ran this Quickstart (day 0). See as example the first admin secret when you make a fresh installation for Jenkins CI.

Regarding rotating secrets (although is a best practice) I believe it is something you need done once you are ready to extend from this blueprint towards your custom blueprint (day 1). There are many other things to do in that list, for example: Deploying the cluster in your own existing VPC or EKS, updating the Authentication Realm, etc.